tfx/benchmarks/benchmark_base.py

```diff
@@ -27,14 +28,73 @@
     "Beam runner to use - any runner name accepted by "
     "apache_beam.runners.create_runner")
 
+modes = ["default", "local_scaled_execution", "cloud_dataflow", "flink_on_k8s"]
```

Comment:
nit: use `ALL_CAPS` for constants as per [PEP-8](https://www.python.org/dev/peps/pep-0008/#constants).

other nit: define these modes separately first
```
DEFAULT_MODE = "default"
LOCAL_SCALED_EXECUTION_MODE = "local_scaled_execution"
modes = [DEFAULT_MODE, LOCAL_SCALED_EXECUTION, ...]
```
then you can used the named constants later.

Reply:
Changed to named constants.